compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/ComposeUIViewControllerConfiguration.uikit.kt

```diff
@@ -66,6 +66,18 @@ class ComposeUIViewControllerConfiguration {
      * explanation on how to fix the issue.
      */
     var enforceStrictPlistSanityCheck: Boolean = true
+
+    /**
+     * If set to true, the Compose will encode the rendering commands on a dedicated render thread,
+     * when possible.
+     * This can improve the performance when no interop UIKit is used.
+     *
+     * It's an experimental API, and the effects of enabling it are not considered stable.
+     *
+     * Changing this setting outside of `ComposeUIViewController` `configure` argument scope has no effect.
+     */
+    @ExperimentalComposeApi
+    var useSeparateRenderThreadWhenPossible: Boolean = false
```

Comment:
I'd like to simplify naming here a bit. WDYT about `parallelRendering`?

Reply:
Sounds good for user-surface API, still want to keep the naming for internal variables where this flag is passed to keep things explicit and concise.